rosdep/python.yaml

```diff
@@ -612,6 +612,33 @@ python-cairo:
     slackpkg:
       packages: [pycairo]
   ubuntu: [python-cairo]
+python-can:
+  alpine:
+    pip:
+      packages: [python-can]
+  arch:
+    pip:
+      packages: [python-can]
+  debian:
+    '*':
+      pip:
+        packages: [python-can]
+    buster: [python-can]
+    sid: [python-can]
+    stretch: [python-can]
+  fedora:
+    pip:
+      packages: [python-can]
+  osx:
+    pip:
+      packages: [python-can]
+  ubuntu:
+    '*':
```

Comment:
Same comment as above, please make the wildcard match the pat package and call out the distros that need the pip version